WebApplication/wwwroot/index.html

```diff
@@ -0,0 +1,66 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+	<title>Home page</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="http://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+	<link rel="stylesheet" type="text/css" href="css/home.css">
+	<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+
+
+</head>
+<body>
+
+    <div id="temp">
+        <nav class="navbar navbar-expand-lg navbar-custom " id="navigationBar">
+            <div class="collapse navbar-collapse " id="navbarSupportedContent">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item active">
+                        <button type="button" class="btn btn-info btn-lg navbar-brand" v-on:click="home">Home</button>
+                    </li>
+                    <li class="nav-item">
+                        <button type="button" class="btn btn-info btn-lg navbar-brand" v-on:click="patient">Patient</button>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <button type="button" class="btn btn-info btn-lg navbar-brand" v-on:click="admin">Admin</button>
+                    </li>
+                    <li class="nav-item">
+                        <h1 class="navbar-brand" id="clinicLogoTxt">Zdravo clinic</h1>
+                        <img src="pictures/logoinvertovanUmanjen.png" id="clinicLogoPic">
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+
+
+
+
+    <div id="routerMode">
+        <router-view></router-view>
+    </div>
+
+
+
+
```

Comment:
It's ok to use just one empty row to separate these parts. It will be easier to maintain if everybody uses the same format.